app/models/contestant.rb

```diff
@@ -1,3 +1,9 @@
 class Contestant < ApplicationRecord
+  has_many :contestant_projects
+  has_many :projects, through: :contestant_projects
 
+
+  def return_projects
+    projects.pluck(:name).join(", ")
```

Comment:
Is this an MVC infraction ? Should I better do the .join(", ") in the view? But if I do so, I just test that part in the view spec?

Reply:
Technically yes, this can go in the view. It's a very minor infraction and I sometimes end up doing the same thing because I like being able to test things at the model level. But when purely sticking to MVC, then yes you can put this in the view and then just test that the display looks right in your feature  spec instead. 

There is always a bit of grey area with MVC when it comes to small stuff like this, which is why we allow for an infraction in ICs and projects without taking points off. We teach pure MVC in mod 2 but then beyond that you will see how these rules are often bent in the real world and you will also start to find occasions where you want to bend the rules yourself!

Reply:
Great work observing the detail in the user story that they should be in a comma separated list :)